server/app/views/admin/programs/ProgramBlockPredicateConfigureView.java

```diff
@@ -745,7 +745,17 @@ private DivTag createValueField(
                     "hidden",
                     "text-xs",
                     BaseStyles.FORM_LABEL_TEXT_COLOR)
-                .withText("Enter a list of comma-separated values. For example, \"v1,v2,v3\"."));
+                .withText("Enter a list of comma-separated values. For example, \"v1,v2,v3\"."))
+        .with(
+            div()
+                .withClasses(
+                    ReferenceClasses.PREDICATE_VALUE_BETWEEN_HELP_TEXT,
+                    "hidden",
+                    "text-xs",
+                    BaseStyles.FORM_LABEL_TEXT_COLOR)
+                .withText(
+                    "Enter two comma-separated or dash-separated values. For example \"18,30\" or"
```

Comment:
Can we describe the inclusive/exclusive logic in help text somewhere? It may not be clear to admins that `BETWEEN` is not inclusive on the upper-bound, or that `AGE_GREATER_THAN` is inclusive.